src/mad_spark_alt/core/terminal_renderer.py

```
@@ -0,0 +1,213 @@
+            self.console.print(content)
+        else:
+            try:
+                markdown = Markdown(content)
+                self.console.print(markdown)
+            except Exception:
+                # Fallback to plain text if markdown rendering fails
+                self.console.print(content)
+    
+    def render_panel(self, content: str, title: str = "", 
+                    border_style: str = "blue", 
+                    title_style: str = "bold") -> None:
```

Ruff (0.12.2)

Unused method argument: `title_style`

(ARG002)